packages/flutter/lib/src/material/time_picker_theme.dart

```diff
@@ -133,117 +162,148 @@ class TimePickerThemeData with Diagnosticable {
   /// theme's [ThemeData.colorScheme].
   final Color? dialTextColor;
 
+  /// The [TextStyle] for the numbers on the time selection dial.
+  ///
+  /// If [dialTextStyle]'s [TextStyle.color] is a [MaterialStateColor], then the
+  /// effective text color can depend on the [MaterialState.selected] state,
+  /// i.e. if the text is selected or not.
+  ///
+  /// If this style is null then the dial's text style is based on the theme's
+  /// [ThemeData.textTheme].
+  final TextStyle? dialTextStyle;
+
+  /// The Material elevation for the time picker dialog.
+  final double? elevation;
+
   /// The color of the entry mode [IconButton].
   ///
   /// If this is null, the time picker defaults to:
   ///
+  ///
   /// ```dart
   /// Theme.of(context).colorScheme.onSurface.withOpacity(
   ///   Theme.of(context).colorScheme.brightness == Brightness.dark ? 1.0 : 0.6,
   /// )
   /// ```
   final Color? entryModeIconColor;
 
-  /// Used to configure the [TextStyle]s for the hour/minute controls.
-  ///
-  /// If this is null, the time picker defaults to the overall theme's
-  /// [TextTheme.headline3].
-  final TextStyle? hourMinuteTextStyle;
-
-  /// Used to configure the [TextStyle]s for the day period control.
-  ///
-  /// If this is null, the time picker defaults to the overall theme's
-  /// [TextTheme.titleMedium].
-  final TextStyle? dayPeriodTextStyle;
+  /// Deprecated. Use [headlineTextStyle] instead.
+  @Deprecated(
+    'Use headlineTextStyle instead. '
+    'This feature was deprecated after v3.6.0-0.0.pre.',
+  )
+  final TextStyle? helpTextStyle;
```

Comment:
Do we have a flutter fix for this and a migration guide? Or do we want to handle that in a separate PR? Or is this an existing deprecation that just got moved around in this cleanup? 😀

Reply:
No, it's a new deprecation, and I think I'd rather handle the flutter fix in another PR.
Or, I could just duplicate the value here, and deprecate it in the PR that has the fix.

Reply:
Actually, I'll just not deprecate it at all in this PR.  We can do that later.

Reply:
Either would be fine. I just wanted to make sure we have a plan to follow through on the deprecation. Thx.